src/js/express.js

```diff
@@ -0,0 +1,148 @@
+// Express instance represented as a component
+//
+// Add additional routers and middleware as child components of this module
+"use strict";
+var fluid       = fluid || require("infusion");
+var namespace   = "gpii.express";
+var express     = fluid.registerNamespace(namespace);
+var path        = require("path");
+var exphbs      = require("express-handlebars");
+var handlebars  = require("handlebars");
+
+express.initExpress = function(that) {
+    if (!that.options.config || !that.options.config.express) {
+        console.error("Cannot initialize express because you have not supplied a 'config' option.");
+        return;
+    }
+
+    // We need these to be model variables so that other grades can listen and take action when they are changed.
+    var express  = require("express");
```

Comment:
It's not possible for the implementation to meaningfully react to changes in these entries without causing widespread corruption. These should just be stored as standard members on the top-level component and documented as fixed. Model material in general should only include those objects which can be expected to survive being serialised to JSON and restored - that is, Strings, Booleans, Numbers, Arrays and Hashes.